transformations/discourse_marker_substitution/transformation.py

```diff
@@ -0,0 +1,142 @@
+import random
+import re
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Substitution of discourse markers with semantically equivalent marker
+"""
+
+CLASS_TO_MARKERS = {
+    "Comparison.Contrast": ["but", "however", "on the contrary"],
+    "Comparison.Contrast.Juxtaposition": ["by comparison", "whereas"],
+    "Contingency.Cause.Reason": ["because", "as", "since", "inasmuch as"],
+    "Contingency.Cause.Result": [
+        "so",
+        "as a result",
+        "consequently",
+        "thus",
+        "therefore",
+        "accordingly",
+        "hence",
+    ],
+    "Expansion.Alternative.Chosen alternative": ["instead"],
```

Comment:
You might want to name this consistent with the others?

Reply:
I'm sorry, what others are you referring to ? I used PDTB classes names